charts/seq/templates/ingress.yaml

```diff
@@ -27,6 +27,9 @@ spec:
       secretName: {{ .secretName }}
 {{- end }}
 {{- end }}
+  {{- if .Values.ingress.ingressClassName }}
+  ingressClassName: {{ .Values.ingress.ingressClassName | quote }}
```

Comment:
Since this lives under the `ingress` object what do you think about just calling this `className`?

Reply:
Hi @KodrAus,

I normally use the same Kubernetes Object name on Helm variables to be clear and easy to understand, this practice is adopted on many public helm charts, like Grafana, Jenkins, etc. But you do you think that className is better, Ok, i change. just confirm please.

Regards

Reply:
I don't think that's a practice we've really followed so far so I think we should go with `className` for now. At some point in the future we can do a complete pass over the API we've evolved here and try make it consistent with patterns used by popular charts.

Reply:
Ok, no problems, change done!